configure_polaris_envs.py

```diff
@@ -102,13 +47,16 @@ def main():
 
     local_mache = args.mache_fork is not None and args.mache_branch is not None
     if local_mache:
-        mache = ''
+        # we will install all packages after cloning the git repo, so no need
+        # to list any here
+        packages = ''
```

Comment:
This still needs 
```suggestion
        packages = '--file deploy/spec-bootstrap.txt'
```
to install the dependencies needed by `bootstrap.py` itself.  Maybe the simpler way to do the whole if/else is:
``` python
    packages = '--file deploy/spec-bootstrap.txt'
    if not local_mache:
        packages = f'"mache={mache_version}" {packages}'

Reply:
Got it, will do!

Reply:
@xylar I just made this change, can you take another look?